### DISCUSSION TITLE
How to define base route to all my routes as vue-router does?

### DISCUSSION THREAD

**Discussion by lucassimines** [nuxt/nuxt#17877] [Questions]
How to define a `base` route to all my routes as vue-router?

If I have an `admin` base for all my routes, then I don't need to add `admin-` before all my `<NuxtLink>` calls.

**Comment by danielroe** [comment]
You can run `NUXT_APP_BASE_URL=/admin yarn nuxi dev` or `NUXT_APP_BASE_URL=/admin node .output/server/index.mjs`
